src/main/web_src/src/components/fagsystem/sykdom/visning/Visning.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import _get from 'lodash/get'
+import Formatters from '~/utils/DataFormatter'
+import SubOverskrift from '~/components/ui/subOverskrift/SubOverskrift'
+import { TitleValue } from '~/components/ui/titleValue/TitleValue'
+import { Bidiagnoser } from './partials/Bidiagnoser'
+import { Perioder } from './partials/Perioder'
+import { ArbeidKodeverk } from '~/config/kodeverk'
+import JournalpostidVisning from '~/components/journalpostid/journalpostidVisning'
+
+interface SykemeldingVisning {
+	data: Array<Sykemelding>
+	ident: string
+}
+
+type Sykemelding = {
+	syntSykemelding: {
+		startDato: string
+		orgnummer: string
+		arbeidsforholdId: string
+	}
+}
+
+type Bestilling = {
+	sykemelding?: Array<Sykemelding>
+}
+
+export const SykemeldingVisning = ({ data, ident }: SykemeldingVisning) => {
```

Comment:
Denne synes jeg ble litt kompleks. Noen mulighet til å dele opp i mindre komponenter?

Hva med en komponetnt for synt og en for detaljert?
syntSykemelding ? <SyntentiskertSykemelding .../> : detaljertSykemelding ? <Sykemlding> : null